src/JBrowse/View/Track/SNPCoverage.js

```diff
@@ -143,14 +147,17 @@ return declare( [WiggleXY, AlignmentsMixin],
             drawRectangle( 'reference', toY( score.total() ), originY-toY( score.get('reference'))+1, fRect);
         });
 
+        var indicator_min_height_prop = this.config.indicatorProp;
+        var indicator_min_height = this.config.indicatorDepth;
+
```

Comment:
Could you change the two variables above to be camelCase?

Reply:
@rbuels  Sure, I can test the functionality

Reply:
Sure, I'll wait until @nathanhaigh pushes the changes so I'm testing the final.

Reply:
Update here in the wiki: http://gmod.org/wiki/JBrowse_Configuration_Guide#SNPCoverage